l10n_nl_banks/data/nl_banks.xml

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<data noupdate="1">
```

Comment:
You can use:


```
<odoo noupdate="1">

```

instead of:

```
<odoo>
	<data noupdate="1">

```

this way you can remove the element `<data>`